.github/workflows/integration.yml

```diff
@@ -139,57 +162,67 @@ jobs:
           - test: TestCpuBoth
             instance: cpu
             failure-prefix: cpu
-          - test: TestGpu
+          - test: TestGpu_g6
             instance: g6
-            failure-prefix: gpu
+            failure-prefix: g6
           - test: TestAarch64
             instance: aarch64
             failure-prefix: aarch64
-          # - test: TestHfHandler
+          # - test: TestHfHandler_g6
           #   instance: g6
           #   failure-prefix: lmi
-          # - test: TestTrtLlmHandler1
+          # - test: TestTrtLlmHandler1_g6
           #   instance: g6
           #   failure-prefix: trtllm
-          # - test: TestTrtLlmHandler2
+          # - test: TestTrtLlmHandler2_g6
           #   instance: g6
           #   failure-prefix: trtllm
-          - test: TestVllm1
+          - test: TestVllm1_g6
             instance: g6
             failure-prefix: lmi
-          - test: TestVllm2
+          - test: TestVllm2_g6
             instance: g6
             failure-prefix: lmi
-          - test: TestVllmCustomHandlers
+          - test: TestVllmCustomHandlers_g6
             instance: g6
             failure-prefix: lmi
-          - test: TestVllmCustomFormatters
+          - test: TestVllmCustomFormatters_g6
             instance: g6
             failure-prefix: lmi
-          - test: TestVllmLora
+          - test: TestVllmLora_g6
             instance: g6
             failure-prefix: lmi
-          - test: TestVllmAsyncLora
+          - test: TestVllmAsyncLora_g6
             instance: g6
             failure-prefix: lmi
-          - test: TestMultiModalVllm
+          - test: TestMultiModalVllm_g6
             instance: g6
             failure-prefix: lmi
-          # - test: TestTextEmbedding
+          # - test: TestTextEmbedding_g6
           #   instance: g6
           #   failure-prefix: lmi
-          # - test: TestCorrectnessTrtLlm
+          # - test: TestCorrectnessTrtLlm_g6
           #   instance: g6
           #   failure-prefix: trtllm
-          - test: TestStatefulModel
+          - test: TestStatefulModel_g6
             instance: g6
             failure-prefix: lmi
+          # P4D instance tests
+          - test: TestVllm_p4d
+            instance: p4d
+            failure-prefix: p4d
```

Comment:
failure prefix need to be lmi. If this test fail, it will not publish lmi image, see https://github.com/deepjavalibrary/djl-serving/blob/master/.github/workflows/nightly.yml#L65-L66